Automattic/wp-calypso
# Gutenboarding: remove page templates

## Changes proposed in this Pull Request

In D45036-code and D46207-code we moved the Gutenboarding page template config JSONs to the backend and ceased pulling them from the frontend. 

This commit deletes the page templates stored on Calypso as they are no longer being used.

## Testing instructions

Head over to `/new` and create a few sites. 

You should see the corresponding page template content for each design in the preview and the created site.
